src/loading2/worker-pool.ts

```diff
@@ -14,14 +16,17 @@ function createWorker(type: WorkerType): Worker {
 	case WorkerType.DECODER_WORKER: {
 		return new DecoderWorker();
 	}
+	case WorkerType.DECODER_WORKER_GLTF: {
+		return new DecoderWorker_GLTF();
```

Comment:
Would it be possible to move the `require` here? That way if client does use only one type of the decoder, they won't over-import.